src/picongpu/CMakeLists.txt

```diff
@@ -200,7 +200,7 @@ add_definitions(-DPIC_VERBOSE_LVL=${PIC_VERBOSE})
 option(PIC_ENABLE_INSITU_VOLVIS "Enable In Situ Volume Visualization" OFF)
 if(PIC_ENABLE_INSITU_VOLVIS)
     add_definitions(-DENABLE_INSITU_VOLVIS=1)
-    set(CUDA_ARCH sm_35 CACHE STRING "set GPU architecture")
+    set(CUDA_ARCH 35 CACHE STRING "Set GPU architecture (semicolon or comma separated list, e.g. '-DCUDA_ARCH=20,35,60')")
```

Comment:
Note to @ax3l: this line should not part of PIConGPU or should trigger an error.
We should we remove  `PIC_ENABLE_INSITU_VOLVIS` because the old insitu code base is not longer compatible with the dev branch of PIConGPU.